src/components/Card/CardItem.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+import { Card } from 'antd';
+
+
+interface Card {
```

Comment:
```suggestion
interface CardProps {
```
просто Card это компонент, а это пропсы компонента